Rules.md

```diff
@@ -84,20 +85,20 @@ brackets ([example]) need no player input and can be done quickly.
 
        *Example: If the dice shows 4 eyes, the player order is Ottoman Empire, Russian Empire, German
        Empire, British Empire, Austro-Hungarian Empire, French Republic.*
-1. **Resolve Invest and Dig trenches tokens** (*)
+1. **Resolve Invest and Dig trenches tokens** (\*)
 1. **Resolve Move tokens** (in player order)
-1. **Production, payment and mutiny** (*)
+1. **Production, payment and mutiny** (\*)
     * **Production:** All players receive the total amount of coins invested in regions they own,
        with a *minimum of 3*.
     * **Payment:** All units in regions the player wants to keep, are paid 1 coin per
        infantry and 2 coins per artillery.
-    * **Mutiny:** All units which were not paid this turn, convert to neutral infantry units (i.e.
+    * **Mutiny:** All units which were not paid this round, convert to neutral infantry units (i.e.
        artillery converts to neutral infantry). Neutral units in sea regions are removed from the
        board.
-1. **[Point scoring, advance turn counter and healing]**
-    * **Point scoring:** Every team scores 1 point per strategic city they own
-    * **Advance turn counter** by one and check the game end condition
-1. **Train units and revival** (*)
+1. **[Point scoring, advance round counter and healing]**
+    * **Point scoring:** Every team scores 1 point per strategic region they own
+    * **Advance round counter** by one and check the game end condition
+1. **Train units and revival** (\*)
```

Comment:
You're escaping the *'s here. I did this with `&#42;` but I think your solution is nicer. Could you make it consistent?